### ISSUE TITLE
Not Found Asset Dependency for AdminLTE

### ISSUE THREAD

**Issue by jjelev** [avanzu/AdminThemeBundle#46]
Hello all this issue relates to issue #6.
The exception which symfony returns is:
[InvalidArgumentException]                                                                                                           
Unable to find file "@AvanzuAdminThemeBundle/Resources/public/vendor/adminlte/plugins/bootstrap-wysihtml5/bootstrap3-wysihtml5.js"

After researching on my own and nothing helped for me ( app/console avanzu:admin:fetch-vendor and even "bower update" in vendor/avanzu/admin-theme-bundle/Resources/bower/ directory) I found this.

I don't know how those dependencies are built but this array is not pointing to the correct file:
```php
'admin_lte_wysiwyg'      => array(
    'inputs' => array(
        $lteJSBase.'plugins/bootstrap-wysihtml5/bootstrap3-wysihtml5.js',
    )
),
```
in vendor/avanzu/admin-theme-bundle/Resources/config/assets.php
The path is correct but filenames are different, should be
`bootstrap3-wysihtml5.all.js`

**Comment by valentinmagrez** [comment]
I m having the same issue, any update soon?

**Comment by jibundeyare** [comment]
check this PR https://github.com/avanzu/AdminThemeBundle/pull/44

**Comment by OskarStark** [comment]
have the same problem
![bildschirmfoto 2015-07-21 um 10 29 24](https://cloud.githubusercontent.com/assets/995707/8796229/6e1a6df6-2f93-11e5-86e1-ffd01736e699.png)


can you please merge PR #44

**Comment by borisschapira** [comment]
Please, could you merge ?

**Comment by aftabnaveed** [comment]
It still seems to be not merged and pushed to packagist.org

**Comment by ogerik** [comment]
+1, please merge

**Comment by aftabnaveed** [comment]
+1 merge please

**Comment by OskarStark** [comment]
@avanzu can you please merge this PR, otherwise the bundle cannot be used.

if there is feedback from your POV, let @jjelev know.

thank you

**Comment by chteuchteu** [comment]
It would be great to add some maintainers to this project so we could merge PRs and push releases

**Comment by senol1** [comment]
I'm in agreement with you

**Comment by avanzu** [comment]
Any volunteers? 
Atm. i just don't have the time to maintain or upgrade this bundle properly.

**Comment by chteuchteu** [comment]
![I volunteer](https://media.giphy.com/media/lu5LYc87R1960/giphy.gif)

Joke aside, I wouldn't have time either to contribute as a main maintainer. But I could help answer issues/categorize them, merge some PRs too
Anyone would be ready to involve himself altogether?

**Comment by OskarStark** [comment]
Trank you @chteuchteu

**Comment by OskarStark** [comment]
Thank*

**Comment by chteuchteu** [comment]
@OskarStark : would it be possible to publish a new release? We're not able to use AdminThemeBundle in its current state.
What about creating a GitHub organization with some maintainers, which would be able to merge PRs, answer & close issues? You would still have full control about what's happening on the project and make releases

What do you think about this?

**Comment by senol1** [comment]
I Think That it an excellent idea and I will be delighted to participated


2015-08-31 15:38 GMT+00:00 Quentin Stoeckel <notifications@github.com>:

> @OskarStark <https://github.com/OskarStark> : would it be possible to
> publish a new release? We're not able to use AdminThemeBundle in its
> current state.
> What about creating a GitHub organization with some maintainers, which
> would be able to merge PRs, answer & close issues? You would still have
> full control about what's happening on the project and make releases
>
> What do you think about this?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/avanzu/AdminThemeBundle/issues/46#issuecomment-136409501>
> .
>



-- 
Sény Mbaye
Etudiant en Master II
Génie Informatique option Génie Logiciel / UFR Sciences et Technologies
Université de THIES
Portable : (+221)775915416
*senyr9@gmail.com <senyr9@gmail.com>*

**Comment by OskarStark** [comment]
@chteuchteu i'm just a user as you, not a maintainer of this repo :)

**Comment by senol1** [comment]
ok I see .... anyway I managed to implement it but I download all of the
assets due myself

2015-08-31 17:57 GMT+00:00 Oskar Stark <notifications@github.com>:

> @chteuchteu <https://github.com/chteuchteu> i'm just a user as you, not a
> maintainer of this repo :)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/avanzu/AdminThemeBundle/issues/46#issuecomment-136445026>
> .
>



-- 
Sény Mbaye
Etudiant en Master II
Génie Informatique option Génie Logiciel / UFR Sciences et Technologies
Université de THIES
Portable : (+221)775915416
*senyr9@gmail.com <senyr9@gmail.com>*

**Comment by chteuchteu** [comment]
@OskarStark Woops sorry, I must has been mislead by one of your previous messages...
@avanzu What do you think?

**Comment by avanzu** [comment]
New patch release is out with the recent fixes. 
Tbh i'm really astonished by the constant interest for this bundle. Thanks for your support. :) 
Anyways, forming an organization might be a bit over the top, collaborators seem to be more appropriate. 

Hairsplitting aside, since @chteuchteu was kind enough to volunteer as tribute, welcome aboard :+1:

**Comment by chteuchteu** [comment]
Thank you @avanzu! I'll try to help regularly, I promise!

**Comment by OskarStark** [comment]
Thank you

Not tried yet but is this Release available via packagist?

Oskar

Am Dienstag, 1. September 2015 schrieb Quentin Stoeckel :

**Comment by daften** [comment]
Yes: https://packagist.org/packages/avanzu/admin-theme-bundle

**Comment by OskarStark** [comment]
great, thank you!

2015-09-01 9:27 GMT+02:00 Dieter Blomme <notifications@github.com>:

**Comment by OskarStark** [comment]
@chteuchteu so can we close this issue then?

**Comment by chteuchteu** [comment]
@OskarStark Yes indeed, it has been fixed by a11a2ee (tested yesterday)
